vector/src/main/java/im/vector/app/features/notifications/NotificationUtils.kt

```diff
@@ -917,7 +916,7 @@ class NotificationUtils @Inject constructor(private val context: Context,
         }
 
         // We cannot use NotificationManagerCompat here.
-        val setting = context.getSystemService<NotificationManager>()!!.currentInterruptionFilter
+        val setting = (context.getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager).currentInterruptionFilter
```

Comment:
Why not fully revert this change?

Reply:
I get a error while compiling letting it as is

Reply:
will revert this